### ISSUE TITLE
Steam: missing libraries causing SteamCMD job to be marked as failed

### ISSUE THREAD

**Issue by philmmanjaro** [ShadowBlip/OpenGamepadUI#196]
OGUI may fail to load steamapi due to missing libs when update function is enabled:

```
Loading Steam API...
dlmopen steamservice.so failed: steamservice.so: cannot open shared object file: No such file or directory 
dlmopen libSDL3.so.0 failed: libSDL3.so.0: cannot open shared object file: No such file or directory
```

**Comment by philmmanjaro** [comment]
A possible workaround can be filtering out the messages: https://github.com/fugasjunior/arma-server-manager/commit/a135b35d36d1252a71a7076e90b09b7263614c6d

**Comment by ShadowApex** [comment]
This issue is tracked here: https://github.com/ShadowBlip/OpenGamepadUI-steam/issues/7

It should be resolved in v1.0.8
